lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
```

Comment:
Use double-quotes for docstrings, per the style guide [1].

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines